.gitignore

```diff
@@ -5,8 +5,8 @@ mstpd
 
 /bridge-stp
 /bridge-stp.tmp
-/lib/ifupdown.sh
```

Comment:
I tend to be neutral too about this, but I can understand the confusion. "lib" (as well as "src") indeed sounds like an internal component, while "utils", "tools" and such sound like scripts, wrappers, mostly optional helpers.

As this commit is properly isolated (thanks Paul, hence can be easily reverted), I'm fine with keeping the change.